.circleci/config.yml

```diff
@@ -158,6 +159,7 @@ commands:
             echo "Using $(pip -V) ($(which pip))"
             pip install tensorboard
             python examples/cifar10.py --lr 0.25 --sigma 0.7 -c 1.5 --sample-rate 0.001 --epochs 1 --data-root runs/cifar10/data --log-dir runs/cifar10/logs --device <<parameters.device>>
+            python -c "import torch; model = torch.load('model_best.pth.tar'); exit(0) if (model['best_acc1']>0.095 and model['best_acc1']<0.12) else exit(1)"
```

Comment:
This doesn't look right - 0.1 accuracy on CIFAR10 is essentially random, so we're not really getting any signal here. We should probably run for few more epochs. We don't need to get to high accuracy (it'll probably take few dozen epochs and doesn't worth it), but something reliably different from random would help. It only takes a few seconds per epoch, so shouldn't be a problem

Reply:
Accuracy remains 0.1 after 10 epochs, creating an issue for that